specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -379,6 +379,54 @@
         }
       }
     },
+    "/{scope}/providers/Microsoft.CostManagement/views/render": {
+      "post": {
+        "tags": [
+          "Views"
+        ],
+        "operationId": "Views_Render",
+        "description": "Renders the view at the given scope and returns the result as a downloadable PNG link.",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/en-us/rest/api/costmanagement/"
+        },
+        "x-ms-examples": {
+          "ResourceGroupViewList": {
+            "$ref": "./examples/ViewRender.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeViewParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/View"
```

Comment:
This should be driven off of the name of the view in the URI since views are a resource that users create.  If you want to allow rendering of a dynamic view without creating a view resource then the action would need to be directly on your provider (i.e. POST .../Microsoft.CostManagement/renderOnDemandView)

Reply:
Change reverted.